exe/Plugins.hs

```diff
@@ -209,8 +214,12 @@ idePlugins recorder includeExamples = pluginDescToIdePlugins allPlugins
     -- The ghcide descriptors should come last so that the notification handlers
     -- (which restart the Shake build) run after everything else
       GhcIde.descriptors pluginRecorder
+#if explicitFixity
```

Comment:
Could you leave a comment about why we want to put this after `GhcIde.descriptors`?